code/src/index.css

```diff
@@ -1,13 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Grape+Nuts&display=swap');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #89a479;
+  color: #000;
+}
+
+main {
+  max-width: 800px;
+  margin: 0 auto;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.custom-checkbox input[type='checkbox'],
+.custom-checkbox .checked {
+  display: none;
+}
+
+.custom-checkbox input[type='checkbox']:checked ~ .checked {
+  display: inline-block;
+}
+
+.custom-checkbox input[type='checkbox']:checked ~ .unchecked {
+  display: none;
+}
+
+/* class for draggable item */
+.dragging {
+  visibility: hidden;
+}
+
+/* class to target the deadline calendar */
+.react-datepicker__input-container input {
+  width: 100%;
+}
+
+input,
+select {
+  background-color: #fff;
+  color: inherit;
+  font-family: inherit;
+  font-size: 100%;
+  border: 1px solid black;
+  border-radius: 5px;
+}
```

Comment:
I can comment all of the styled components and ".css" on this one.
really good design!